### ISSUE TITLE
Jamtaba2: src/hostapi/alsa/pa_linux_alsa.c:3636: PaAlsaStreamComponent_BeginPolling: Assertion `ret == self->nfds' failed.

### ISSUE THREAD

**Issue by brylie** [elieserdejesus/JamTaba#1202]
When starting JamTaba with JACK running, it crashes with the following error:

> Jamtaba2: src/hostapi/alsa/pa_linux_alsa.c:3636: PaAlsaStreamComponent_BeginPolling: Assertion `ret == self->nfds' failed.

**Comment by elieserdejesus** [comment]
@brylie , I`m not a Linux user, so I never understand how jack works in Linuc. Can you explain how jack is used by musicians? Jack is something like "asio" for linux?

**Comment by brylie** [comment]
Yeah, JACK is basically an audio layer that lets you send audio/midi between various apps.

**Comment by pljones** [comment]
That's the portaudio code...  It's also saying alsa, rather than jack.  So are you sure you've got the settings right?  I'm guessing it could be something like if JACK's got exclusive access to your soundcard, ALSA won't be able to open it.

**Comment by brylie** [comment]
Hm, good point. I'm not sure how to run JACK without an exclusive lock. In any case, I can run JamTaba when JACK is not running.

**Comment by pljones** [comment]
Actually I think I had it upside down - JACK's using ALSA to access the sound card.  Too long since I gave up trying to do audio on Linux...  I'll shut up :)

**Comment by raymond-lemarron** [comment]
> @brylie , I`m not a Linux user, so I never understand how jack works in Linuc. Can you explain how jack is used by musicians? Jack is something like "asio" for linux?

if you want to know more on JACK read this document. It is powerfull and used by many software : guitarix (guitar effects), cadence, hydrogen, and several multiplatform DAWs like Ardour, LMMS and obviously REAPER
